python/lsst/pipe/analysis/plotUtils.py

```diff
@@ -344,6 +363,7 @@ def bboxToRaDec(bbox, wcs):
     ra, dec = zip(*corners)
```

Comment:
Two lines above here (line 361: `coord = wcs.pixelToSky(p).toIcrs()`)  needs to be fixed too